x11-misc/mugshot/mugshot-0.4.0.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1 eapi7-ver gnome2-utils
+
+MY_PV="$(ver_cut 1-2)"
+
+DESCRIPTION="A lightweight user-configuration application"
+HOMEPAGE="https://launchpad.net/mugshot"
+SRC_URI="https://launchpad.net/${PN}/${MY_PV}/${PV}/+download/${PN}-${PV}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="gnome libreoffice pidgin webcam"
+
+RDEPEND="dev-python/dbus-python[${PYTHON_USEDEP}]
+	dev-python/pexpect[${PYTHON_USEDEP}]
+	dev-python/pycairo[${PYTHON_USEDEP}]
+	dev-python/pygobject:3[${PYTHON_USEDEP}]
+	sys-apps/accountsservice
+	x11-libs/gtk+:3
```

Comment:
Don't you need introspection?

Reply:
Seems so, when I remove it, mugshot won't start and complains. Added.